src/daft-connect/src/session.rs

```diff
@@ -0,0 +1,48 @@
+use std::collections::{BTreeMap, HashMap};
+
+use uuid::Uuid;
+
+pub struct Session {
```

Comment:
Since we know SQL will need a session at some point, it makes sense to have this in a shared crate